dataland-frontend/tests/e2e/specs/user-api-key/RevokeApiKey.ts

```diff
@@ -0,0 +1,16 @@
+describe("As a user I expect my api key will be revoke correctly", () => {
```

Comment:
This test will only run succesfully if 
`As a user I expect my api key will be generate correctly` did run before. There should not be dependencies between tests - hence please put generation and revoke into a single test.

Reply:
The intention is for them to work separately. That's why we simulate the response from the server that the code exists, it sets the application according to the code from the fixture: "ApiKeyInfoMockWithKey.json" and sends a removal request. The test testing the removal method itself can be found in the backend tests.

Reply:
Got it. Thank you for the eplanation. 
In that case pleas only corret the title/description (`it("...")`)

Reply:
ok